colibre/auto_plotter/cold_gas_fractions.yml

```diff
@@ -0,0 +1,128 @@
+cold_gas_frac_func_stellar_mass:
+  type: "scatter"
+  legend_loc: "lower left"
+  comment: "$10^9 M_{\\odot} < M_{\\ast} < 10^{11.5} M_{\\odot}$"
+  x:
+    quantity: "apertures.mass_star_100_kpc"
+    units: Solar_Mass
+    start: 2e7
+    end: 1e12
+  y:
+    quantity: "derived_quantities.neutral_to_stellar_mass_100_kpc"
+    units: "Solar_Mass / Solar_Mass"
+    start: 1e-4
+    end: 100
+  median:
+    plot: true
+    adaptive: true
+    log: true
+    number_of_bins: 15
+    start:
+      value: 2e7
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-Cold gas Fraction
+    section: Cold Gas Data Comparison
+    caption: Galaxy H_$2$ mass (including Helium following the observations) over stellar mass as a function of stellar mass in 0.2 dex bins, measured in 100 kpc apertures
```

Comment:
Galaxy H_$2$  -> Galaxy H$_2$?

Also, it looks like the bins here are not 0.2 dex